portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,70 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Jonathan Ward's Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <h1 style="background-color:DodgerBlue;color:White;text-align:center">Jonathan Ward's Portfolio</h1>
```

Comment:
Instead of writing all your style inline, add it to your css file as a class or id and set it on your elements.

**`style.css file`**
.title {
    background-color: DodgerBlue;
    color: White;
    text-align: center;
}

**index.html**
h1 class="title"...